caesars-palace/blocks/slider-card/slider-card.css

```diff
@@ -90,18 +108,18 @@
     display: block;
     font-family: var(--text-font-family);
     font-weight: var(--emphasized-font-weight);
-    font-size: 18px;
+    font-size: 16px;
```

Comment:
I think there is a variable for that size

Reply:
It depends on what you are trying to do. There is a `--body-font-size-s` var which is mobile aware. It is 14px on small screens and then bumps up to 16px on larger screens. If you want to force your font-size to always be 16px it is better to hard code it like you have done.